composer.json

```diff
@@ -14,7 +14,7 @@
         "php": "^7.1",
         "doctrine/dbal": "^2.6",
         "ocramius/proxy-manager": "^2.0.2",
-        "symfony/console": "^4.0"
+        "symfony/console": "^3.4|^4.0"
```

Comment:
I suggest using `||` as the OR operator rather than the older `|` to follow the composer recommendation (choosing the double pipe as the main operator was done because it is also the operator used by ruby's bundler and by npm, making the syntax consistent between package managers)

Reply:
Doctrine uses single pipe everywhere currently. Could you point to the _recommendation_ and ideally to _deprecation_ of current format?

Reply:
Updated.

Reply:
deprecation is not planned (it would break all existing releases using it, so it is almost impossible to deprecate anything there). But the older operators have been removed from the composer doc in favor of new ones (this was done when updating the doc in https://github.com/composer/composer/pull/2596/commits/ebab9db27ac3887babc8404107dd448d974b2f43)